CHANGELOG.md

```diff
@@ -6,6 +6,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- more Task Enum tasks
+- accelerator options
+- batch_size and hardware suggestion
+- ram_size_mb to specify model ram requirements during inference
+- added time to the Tensor object as an optional dim
+- Use common metadata Asset Object to refer to Model Artifact and artifact metadata as a Collection level object
+
+### Changed
+- selected_bands > band_names, the same human readable names used in the common metadata band objects.
```

Comment:
One important distinction of `selected_bands` was that it indicated the bands selected for the model inputs, which might not be all the available bands from the reference EO imagery. This needs to be cleared up in the README. I also saw that `Band` object was used directly in the README. Is the "human readable names" mention here still up to date (it makes sense to be to use only the names, but just to make sure).

Reply:
Sorry I missed that in the Changlog, I removed band_names and instead used a list of band objects

> We use the [STAC 1.1 Bands Object](https://github.com/radiantearth/stac-spec/pull/1254) for representing bands information, including nodata value, data type, and common band names. Only bands used to train or fine tune the model should be included in this list.

I saw you commented earlier that instead of using a Band object in the extension we can just use the band name from the field of the Band objects used to train the model. That sounds good to me.